packages/react-scripts/backpack-addons/sassFunctions.js

```diff
@@ -3,10 +3,12 @@ const sass = require('node-sass');
 
 module.exports = {
   sassOptions: {
-    'encodebase64($string)': str => {
-      const buffer = Buffer.from(str.getValue());
-
-      return sass.types.String(buffer.toString('base64'));
+    functions: {
```

Comment:
It is re-build result in banana:
![image](https://user-images.githubusercontent.com/86700834/156288982-51e34ded-ca51-4af0-8152-4ce8ff865cec.png)